app/Espo/ORM/Repositories/RDB.php

```diff
@@ -192,7 +192,7 @@ public function save(Entity $entity, array $options = [])
             try {
                 $result = $this->getMapper()->insert($entity, !empty($options['ignoreDuplicate']));
             } catch (UniqueConstraintViolationException $e) {
-                throw new NotUnique('The record cannot be created due to database constraints.');
+                throw new NotUnique($e->getMessage());
```

Comment:
Please. change to.
throw new NotUnique('The record cannot be created due to database constraints. Error: ' . $e->getMessage());